src/network/networkd-can.c

```diff
@@ -175,13 +175,13 @@ static int link_set_can(Link *link) {
         if (link->network->can_fd_mode >= 0) {
                 cm.mask |= CAN_CTRLMODE_FD;
                 SET_FLAG(cm.flags, CAN_CTRLMODE_FD, link->network->can_fd_mode);
-                log_link_debug(link, "%sabling FD mode", link->network->can_fd_mode ? "En" : "Dis");
+                log_link_debug(link, "Setting FD mode to '%sd'.", enable_disable(link->network->can_fd_mode));
```

Comment:
I think `yes_no()` or `true_false()` is better. Then we can simply use the string as is, that is, without `d`.

Reply:
Okay, I went for `yes_no()` and force-pushed again.